pkg/liqo-controller-manager/resource-request-controller/offerUpdater.go

```diff
@@ -92,13 +92,14 @@ func (u *OfferUpdater) CreateOrUpdateOffer(cluster discoveryv1alpha1.ClusterIden
 	if request == nil {
 		// invalid clusterID so return requeue = false. The clusterID will be removed from the workqueue and
 		// the resourcereader (in a daisy chain if there are multiple).
+		// TODO: SHOULD I REQUEUE IF AN ERROR OCCUR?
```

Comment:
I suppose you refer to the modification below. If so, yes, as in case of temporary errors it makes sense to retry.